.circleci/config.yml.in

```diff
@@ -127,9 +127,10 @@ jobs:
       - checkout
       - run:
           command: |
-            sudo apt-get update -y
-            sudo apt-get install -y clang-format
-            ./travis-scripts/run-clang-format/run-clang-format.py -r torchvision/csrc
+            curl https://oss-clang-format.s3.us-east-2.amazonaws.com/linux64/clang-format-linux64 -o clang-format
+            chmod +x clang-format
+            sudo mv clang-format /opt/clang-format
```

Comment:
nit: do we need to move this to `/opt/clang-format` if we can specify a `--clang-format-executable` in the script?

Reply:
It's not really necessary. I did not want to leave a binary in the root of the project. Can make a change if you want.

Reply:
No worries, it was just for my information.